test-assets/role-manifests/kube/volumes-with-annotation.yml

```diff
@@ -0,0 +1,35 @@
+---
+roles:
+- name: myrole
+  jobs:
+  - name: tor
+    release_name: tor
+  run:
+    capabilities:
+    - something
+    scaling:
+      min: 1
+      max: 2
+    persistent-volumes:
```

Comment:
This should probably be gradually moved to:
```yaml
volumes:
- path: /mnt/persistent
  type: persistent
  …
- path: /mnt/shared
  type: shared
  …
- path: /sys/fs/cgroup
  type: host
  …
```

Reply:
I dont follow you here.

Reply:
We're basically keeping the `persistent-volumes` and `shared-volumes` keys for backwards compatibility; we're trying to move towards `volumes` with individual types to have more uniform support for `host` and `empty` volume mount types.  So having the backwards compatible forms in the test cases doesn't seem to make much sense.

Reply:
ok, I got it. I think I start the PR based on the [postgres role](https://github.com/SUSE/scf/blob/develop/container-host-files/etc/scf/config/role-manifest.yml#L386)  and that did not help in considering the new `volumes` setup. Any reasons why postgres still using `persistent-volumes` key?